test/action.test.js

```diff
@@ -263,3 +263,29 @@ tap.test('should call external api for github-action-merge-dependabot major rele
   t.ok(stubs.logStub.logWarning.calledOnce)
   t.ok(stubs.fetchStub.calledOnce)
 })
+
+tap.test('should check submodules semver when target is set', async t => {
```

Comment:
this is not what submodules look like. submodules have a commit sha in the pr title (iirc) https://github.com/fastify/github-action-merge-dependabot/issues/95

Reply:
@Eomm lmk if the comment is clear. It's about the title of the PR you have in this test not matching what happens when a version of a submodule tries to be bumped. You only get a SHA in that case

Reply:
Yeah, I'm on it. Let me push the wip code

I'm trying to ignore the case:

- the PR title is not a semver and the user set the `target` value

this check is a bit tricky due the `prerelease` stuff and the coercion.

Reply:
we don't have to support _everything_, it would be a good step if we didn't break anything of the existing and we document the behavior for edge cases so we know what we can work on next

Reply:
I have implemented the most simple check, using this PR as reference:

https://github.com/sitiom/dotfiles/pull/1